cms/templates/index.html

```diff
@@ -302,9 +302,9 @@ <h3 class="course-title">${course_info['display_name']}</h3>
       <div class="notice notice-incontext notice-instruction notice-instruction-nocourses list-notices courses-tab active">
         <div class="notice-item">
           <div class="msg">
-            <h3 class="title">${_("Are you staff on an existing {studio_name} course?").format(studio_name=settings.STUDIO_SHORT_NAME)}</h3>
+            <h3 class="title">${_("Are you staff on an existing {studio_name} course?").format(studio_name=set)}</h3>
```

Comment:
Why was `settings.STUDIO_SHORT_NAME` changed `set`. I assume this is a typo. This resulted in [TNL-4167](https://openedx.atlassian.net/browse/TNL-4167)